test/models/mnist/validate_mnist.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+mnist_training_example="../../../examples/mnist/mnist_deep_simplified.py"
+echo $mnist_training_example
+
+echo "**************************************************************************"
+echo "Run MNIST ADAM OPTIMIZER on NGRAPH"
+echo "**************************************************************************"
+python $mnist_training_example --train_loop_count=20 --make_deterministic
+
+echo "**************************************************************************"
+echo "Run MNIST ADAM OPTIMIZER on TF"
+echo "**************************************************************************"
+NGRAPH_TF_DISABLE=1 python $mnist_training_example --train_loop_count=20 --make_deterministic
+
+echo "**************************************************************************"
+echo "Run MNIST GRADIENT DESCENT on NGRAPH"
+echo "**************************************************************************"
+
+python $mnist_training_example --train_loop_count=20 --make_deterministic --optimizer sgd
+
+echo "**************************************************************************"
+echo "Run MNIST GRADIENT DESCENT on TF"
+echo "**************************************************************************"
+
+NGRAPH_TF_DISABLE=1 python $mnist_training_example --train_loop_count=20 --make_deterministic --optimizer sgd
```

Comment:
Currently this is something you can quickly run and verify the accuracy that gets printed. I will change this to an automated test in another PR. It's a handy script for now.